### PR TITLE
CLI README Cleanup

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves [742](https://github.com/astronomer/docs/issues/742)

### CODE REVIEW

**Comment by kwcanuck** [comment]
@jwitz a preliminary draft is ready for your review.

File: README.md
Comment (jwitz) [inline README.md:196]:
```suggestion
    `brew install golangci-lint`
```
Either have these in single tick fences or indent them so that the fences look like how we have them in Astro and Software docs.

File: README.md
Comment (jwitz) [inline README.md:190]: It would be great if we could remove everything about local development to a new `CONTRIBUTING.md` doc at the top level of the repo. This content is primarily for Astronomer developers, and I don't want people to mistake it for "Airflow project development"

File: README.md
Comment (neel-astro) [reply README.md:196]: Three tick fences are supposed to represent a code block in Github, which allows someone to easily copy and paste the code block.

File: README.md
Comment (neel-astro) [inline README.md:253]:
```suggestion
To generate mocks for a new interface, add the following command below the `mock` rule in `Makefile` and then run `make mock`:
```

File: README.md
Comment (neel-astro) [inline README.md:306]: This isn't relevant to the Astro CLI project, it should rather be part of the `astronomer/docs` repo, at best if you want you can add these sections to the `astronomer/docs` repo and leave a link to that over here.

File: README.md
Comment (neel-astro) [reply README.md:190]: If we are doing this then we should also move the "Run Tests" & "Generate Mocks" sections to `CONTRIBUTING.md` as well

**Review by neel-astro — CHANGES_REQUESTED** [review]

**Review by neel-astro — COMMENTED** [review]
Love the effort ❤️

File: README.md
Comment (kwcanuck) [reply README.md:11]: @jwitz I had the same thought about a table. However, I thought a table made the read me too visually cluttered. This simple list seems to work best. I don't feel strongly about it. I'll add the table if you think it works better.

File: README.md
Comment (kwcanuck) [reply README.md:64]: @jwitz and @neel-astro I'll keep the standard Windows procedure as it seems that it's necessary for some users.

File: README.md
Comment (kwcanuck) [reply README.md:306]: @neel-astro @paolaperaza requested that we add this information to the README. You can review her comment in the originating issue. This information is a duplicate of what already appears in the astronomer/docs repo. I'll remove it.

File: README.md
Comment (kwcanuck) [reply README.md]: @jwitz and @neel-astro I have created the recommended CONTRIBUTING.md doc. I'm not sure how to configure it properly, or make sure it appears "at the top level of the repo". I'll leave it to you to make the necessary adjustments.

**Comment by kwcanuck** [comment]
@jwitz and @neel-astro this PR is ready for your secondary review.

File: README.md
Comment (neel-astro) [reply README.md]: ideally, Astro CLI contribute section should have information on how to contribute to the Astro CLI repo only, not sure if @paolaperaza also meant the same thing in the original issue description

**Review by neel-astro — APPROVED** [review]

File: README.md
Comment (kushalmalani) [inline README.md:41]: Should this be v0.13.1 ?

File: README.md
Comment (kushalmalani) [inline README.md:50]: Same as the above comment

**Review by kushalmalani — APPROVED** [review]

File: README.md
Comment (kwcanuck) [reply README.md:41]: @kushalmalani the version information is correct:

image

File: README.md
Comment (kwcanuck) [reply README.md:50]: @kushalmalani as noted in the previous response, the version information is correct.

**Comment by kwcanuck** [comment]
@neel-astro or @kushalmalani can you merge this PR when time permits? Thanks!

**Pull request merged**
